docker-compose/traefik/ucentral.yaml

```diff
@@ -0,0 +1,77 @@
+http:
```

Comment:
Would've preferred to [configure traefik using labels](https://doc.traefik.io/traefik/user-guides/docker-compose/basic-example/) but as this works as well I'm totally fine with it.

Reply:
yes I also deployed it by configuring labels but since this requires mounting the Docker socket I decided against it. Maybe there is going to be some development on this in the future like finer-grained access control regarding the Docker socket (honestly I wonder why nothing happened on that front at all), so I'll definitely keep an eye out.